### PR TITLE
fixed hardcoded room-code issue

### PR DESCRIPTION
**Pull request by onesquarepeg**
Made a const for length of the room code. And made a util func to generate room code that will generate code based on the set length.
By the way, I set the length of the room code to 4. :)

### CODE REVIEW

File: server/cmd/utils/utils.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"math"
+	"math/rand"
+	"time"
+)
+
+const LENGTH_OF_ROOM_CODE = 4
```
Comment (tusharsadhwani) [inline server/cmd/utils/utils.go]: use camel case, `RoomCodeLength`

File: server/cmd/utils/utils.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+func GenerateRoomCode() int {
+	rand.Seed(time.Hour.Nanoseconds())
```
Comment (tusharsadhwani) [inline server/cmd/utils/utils.go]: you're only supposed to seed once, make a seeded random number generator using

```go
var seed = rand.NewSource(time.Now().UnixNano())
var rng = rand.New(seed)
```

in global scope instead

File: server/cmd/utils/utils.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	roomCode := 0
+	for i := LENGTH_OF_ROOM_CODE - 1; i >= 0; i-- {
```
Comment (tusharsadhwani) [inline server/cmd/utils/utils.go]: you can do this without a loop

```go
lowerLimit := int(math.Pow10(RoomCodeLength - 1))
upperLimit := int(math.Pow10(RoomCodeLength))
roomCode := lowerLimit + rng.Intn(upperLimit - lowerLimit)
```

**Review by tusharsadhwani — CHANGES_REQUESTED** [review]

File: server/cmd/utils/utils.go (hunk 3 above)
Comment (onesquarepeg) [reply server/cmd/utils/utils.go]: Applied all the changes, re-review please!

**Review by tusharsadhwani — APPROVED** [review]
LGTM

**Pull request merged**